app/views/drivers/_form.html.slim

```diff
@@ -1,4 +1,5 @@
-= form_with model: driver do |f|
+= form_with model: driver, local: true do |f|
+  = render partial: 'shared/error', locals: { models: @driver, data_collection: drivers }
```

Comment:
whats the `data_collection being` used for? we dont use that variable so you can remove it
`models` its singular so it shouldnt be a plural variable imo, can we just set this as `model` if that doesnt break anything.
also dont use `@` use your helper method that sets it up like alex got you to do, so use `driver`

```suggestion
  = render partial: 'shared/error', locals: { model: driver }
```